### PR TITLE
feat: add getting started section

### PR DESCRIPTION
**Pull request by Wodann**

### CODE REVIEW

**Comment by Wodann** [comment]
[How to use runtime docs](https://trello.com/c/tKJOSCvB/84-how-to-use-runtime-docs)

File: src/ch01-02-hello-hot-reloading.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Mun embedded in C++
+
+Mun [exposes](github.com/mun-lang/runtime-ffi) a C API and complementary c++ bindings for the Mun
+Runtime. Listing 1-3 shows a C++ application that constructs a Mun Runtime for the
+`hello_fibonacci` library and continuously invokes the `fibonacci5` function and outputs its result.
+
+Filename: main.cc
+
+```cpp
```
Comment (baszalmstra) [inline src/ch01-02-hello-hot-reloading.md:36]: Hightlighting of cpp doesnt seem to work..

**Review by baszalmstra — APPROVED** [review]
Very nicely written

File: src/ch01-02-hello-hot-reloading.md (hunk 1 above)
Comment (Wodann) [reply src/ch01-02-hello-hot-reloading.md:36]: Fixed. It was missing a closing **"**

**Pull request merged**